packages/strapi-design-system/vite.config.ts

```diff
@@ -15,7 +17,7 @@ export default defineConfig(async () => {
       exclude: [],
     },
     build: {
-      emptyOutDir: false,
+      emptyOutDir: env.DTS === 'true',
```

Comment:
How come we needed to do this?

Reply:
I added this line of code because in the previous version, the value was set to false. Every time I ran "yarn build" after running "DTS=true yarn build", the dist folder would contain the old build result with the environmental variable. Therefore, my goal is to always have a completely new version in the dist folder when I run "DTS=true yarn build"